packages/storybook/stories/components/Textarea.stories.tsx

```diff
@@ -0,0 +1,173 @@
+/* @license CC0-1.0 */
+
+import type { Meta, StoryObj } from '@storybook/react';
+import { Textarea } from '@utrecht/component-library-react/src/css-module/Textarea';
+import readme from '@utrecht/components/textarea/README.md?raw';
+import tokensDefinition from '@utrecht/components/textarea/tokens.json';
+import tokens from '@utrecht/design-tokens/dist/index.json';
+import React from 'react';
+import { designTokenStory } from './util';
+
+const defaultArgs = {
+  dir: 'auto',
+  disabled: false,
+  focus: false,
+  hover: false,
+  invalid: false,
+  placeholder: undefined,
+  readOnly: false,
+  required: false,
+  spellCheck: false,
+  children: undefined,
+};
+
+const exampleArgs = {
+  children: 'The Quick Brown Fox Jumps Over The Lazy Dog',
+};
+
+const arabicDecorator = (Story) => (
+  <div dir="rtl" lang="ar">
+    {Story()}
+  </div>
+);
+
+const meta = {
+  title: 'CSS Component/Textarea',
+  id: 'css-textarea',
+  component: Textarea,
+  argTypes: {
+    disabled: {
+      description: 'Disabled',
+      control: 'boolean',
+    },
+    invalid: {
+      description: 'Invalid',
+      control: 'boolean',
+    },
+    readOnly: {
+      description: 'Read-only',
+      control: 'boolean',
+    },
+    required: {
+      description: 'Required',
+      control: 'boolean',
+    },
```

Comment:
```
  spellcheck: {
    description: 'Spellcheck',
    control: { type: 'select' },
    options: ['', 'false', 'true'],
  },
```

The existing code includes the above definition for `spellcheck`, because spell check is not a boolean. To quote MDN:

> It may have the following values:
> 
> empty string or `true`, which indicates that the element should be, if possible, checked for spelling errors;
> 
> `false`, which indicates that the element should not be checked for spelling errors.

The default value is now `false`, which means all textareas have spellchecking disabled, which is not desired. Also, the args table interaction has changed unfavourably:

## Main branch

<img width="767" alt="Screenshot 2023-08-29 at 09 15 14" src="https://github.com/nl-design-system/utrecht/assets/30694/b5967c2c-0e99-4772-a629-a557cac08736">

## This branch

<img width="699" alt="Screenshot 2023-08-29 at 09 15 23" src="https://github.com/nl-design-system/utrecht/assets/30694/be193a27-f491-4cc6-b93c-921246428f80">

I reckon it is best to keep the original arg type for this property.